docs/Contributing/Seeding-Data.md

```diff
@@ -2,47 +2,80 @@
 
 When developing Fleet, it may be useful to create seed data that includes users and teams.
 
+
+> In order to run scripts that make use of premium features, make sure you have started the server with the correct flags as described in [Testing](./testing-and-local-development.md#license-key).
+
 Check out this Loom demo video that walks through creating teams seed data:
 https://www.loom.com/share/1c41a1540e8f41328a7a6cfc56ad0a01
 
-For a text-based walkthrough, check out the following steps:
+For a text-based walkthrough, follow these steps:
+
+## Create an environment variable file
 
 First, create an `env` file with the following contents:
 
 ```
 export SERVER_URL=https://localhost:8080 # your Fleet server URL and port
 export CURL_FLAGS='-k -s' # set insecure flag
-export TOKEN=eyJhbGciOi... # your login token
+export TOKEN=eyJhbGciOi... # your api token
 ```
 
-Next, set the `FLEET_ENV_PATH` to point to the `env` file. This will let the scripts in the `fleet/` folder source the env file.
+Conveniently, you can store the `env` file as `fleet_env` in the `./code/fleet-api-env` directory.
+
+## Apply your environment variable file
+
+Next, set the `FLEET_ENV_PATH` to point to the `env` file using the command line.
 
 ```
-export FLEET_ENV_PATH=/Users/victor/fleet_env
+export FLEET_ENV_PATH=./code/fleet-api-env/fleet_env
 ```
 
-Finally, run one of the bash scripts located in the [/tools/api](../../tools/api/README.md) directory.
+This will let the scripts in the `/tools/api/fleet/` folder source the `env` file.
+
+## Run one of the bash scripts to seed the data
+
+Finally, run one of the bash scripts located in the [/tools/api](../../tools/api) directory.
```

Comment:
This seems to redirect to a 404. I don't think we can have relative paths in our docs anymore; it has to be absolute.

Reply:
Er I should say, this would work in GitHub but not the website. For the website to work, it needs to be an absolute link to that directory in github.

Reply:
sgtm, going to get in these changes this morning